sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/_transport.py

```diff
@@ -503,13 +503,15 @@ def __init__(
         self.sslopts = ssl_opts if isinstance(ssl_opts, dict) else {}
         self.sslopts['server_hostname'] = host
         self._read_buffer = BytesIO()
+        self._use_tls = kwargs.get("use_tls")
```

Comment:
lets make this a typed kwarg at the `AbstractTransport` and then bring it to the implementors as well